README.md

```diff
@@ -15,4 +54,14 @@ $ pip install -r requirements.txt
 If something goes wrong, you can always:
 ```bash
 $ rm -rf $venv_path
-```
\ No newline at end of file
+```
+
+## Roundtrip tests
+The roundtrip tests hit the Neptune backend directly, which requires a valid Neptune configuration. As amundsen-gremlin CI does not currently have AWS configured, these tests do not run by default.
+
+In order to run the roundtrip tests:
+```bash
+$ python -m pytest --roundtrip .
```

Comment:
Is this the pref. way to run pytest? I typically just do `pytest ....`. Is there a difference? Not important, just curious :-)

Reply:
I think depending on your environment it should be aliased to pytest, I'm just used to using it this way, I doubt it's any better. `pytest` is more concise though, so I'll fix it.